R/DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: apache.sedona
 Title: R Interface for Apache Sedona
-Version: 1.3.1
```

Comment:
Is there a reason why you use 1.3.1.9000?

Reply:
R Tidyverse package conventions: 1.3.1 is the latest CRAN version, 1.3.1.9000 is the dev version (similar to 1.4.0-SNAPSHOT) and it will be replaced with 1.4.0 for the release.
https://r-pkgs.org/lifecycle.html#sec-lifecycle-version-number-tidyverse